html/infrastructure/safe-passing-of-structured-data/structured_clone_bigint.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html>
+ <head>
+  <meta content="text/html; charset=utf-8" http-equiv="content-type" />
+  <title>2.7 Safe passing of structured data</title>
+  <link rel="help" href="http://www.w3.org/TR/html5/common-dom-interfaces.html#safe-passing-of-structured-data" />
```

Comment:
Fix this to be https://html.spec.whatwg.org/multipage/#safe-passing-of-structured-data

Reply:
Done